### ISSUE TITLE
Slower Speed While Training

### ISSUE THREAD

**Issue by Ming-er** [SWivid/F5-TTS#969]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Question details

Hi, authors. Previously, I have tried to train F5-TTS on a relatively small dataset (LibriTTS). Recently, I switch to train it on an Emilia-EN sub set (about 9.5k hours extracted from the full dataset) while leaving other settings unchanged. However, I found that the training speed has slowed down to about 50% compared to the training process on LibriTTS. Is it normal?

**Comment by SWivid** [comment]
I think it's normal cuz Emilia samples are up to 30s, longer than LibriTTS ones.

**Comment by Ming-er** [comment]
Sorry to bother again. I still have a question during traning. I found that the model trained on Emilia-EN-char subset (mentioned above, containing about 9.5k hours from en subset of Emilia) converged much slower than trained on LibriTTS. For example, both trained using 100k steps, the wer of the former is much higher than the later. Is it normal? What may be the main reason? I'll appreciate your reply, thank you ~

**Comment by SWivid** [comment]
> For example, both trained using 100k steps, the wer of the former is much higher than the later

How much slower

**Comment by Ming-er** [comment]
> > For example, both trained using 100k steps, the wer of the former is much higher than the later
> 
> How much slower

The model trained on the Emilia-EN subset may require 200k steps to achieve similar wer or spk-sim with the model trained on the LibriTTS (100k steps)

**Comment by Ming-er** [comment]
> > > For example, both trained using 100k steps, the wer of the former is much higher than the later
> > 
> > 
> > How much slower
> 
> The model trained on the Emilia-EN subset may require 200k steps to achieve similar wer or spk-sim with the model trained on the LibriTTS (100k steps)

All other settings are kept same, including the total training steps, batch size, max samples, lr and so on.

**Comment by Ming-er** [comment]
> > The model trained on the Emilia-EN subset may require 200k steps to achieve similar wer or spk-sim with the model trained on the LibriTTS (100k steps)
> 
> have you evaluated on librispeech test-clean or seed-tts test-en, former one is of same domain to libritts

I evaluated on ls-pc-test-clean, seedtts-en as well as ellav-hard. However, similar trends are observed during training

**Comment by SWivid** [comment]
how about spk-sim

**Comment by SWivid** [comment]
in our expr and also the compr mentioned in previous issue, the wer of small model trained on libritts even higher than base model trained on emilia, could interpreted as libritts is very clean but emilia is wild
but consider spk-sim, in our expr, train with emilia has a lot of gain

**Comment by Ming-er** [comment]
> in our expr and also the compr mentioned in previous issue, the wer of small model trained on libritts even higher than base model trained on emilia, could interpreted as libritts is very clean but emilia is wild
> 
> but consider spk-sim, in our expr, train with emilia has a lot of gain
> 
> observation mentioned above are based on eval results on and after 400k, we eval relatively fewer with early step ckpts

OK, thank you for your guidance. I will keep monitoring the later training progress.
